.github/workflows/ci-e2e.yml

```diff
@@ -58,32 +58,36 @@ jobs:
             - name: Check out
               uses: actions/checkout@v3
 
-            - name: Group spec files into chunks of three
+            - name: Pick 4 random spec files
```

Comment:
I do think once a day we should run them all and send an event to posthog on failure so we can alert from CDP / track incidence

but otherwise 100

these tests are slow and low value